### ISSUE TITLE
Conv2d cores `independent_bn_bias` parameter not longer required

### ISSUE THREAD

**Issue by MaxFBurg** [sinzlab/neuralpredictors#219]
The parameter `independent_bn_bias` ( https://github.com/sinzlab/neuralpredictors/blob/29206ece4ed20af57f7c6e5ee614e49da50dd8d5/neuralpredictors/layers/cores/conv2d.py#L60
 ) in the conv2d cores is not longer required, as now batchnorm scale and bias parameters can be precisely controlled by passing a list of booleans to `batch_norm`, `batch_norm_scale`, and `bias`. 

`independent_bn_bias` was so far kept for backwards compatibility. Now its obsolete and could be removed in the future for the next version.

**Comment by pollytur** [comment]
Solved in #221
